### DISCUSSION TITLE
Localization - Can I help add Afrikaans

### DISCUSSION THREAD

**Discussion by JonathanAspeling** [alisonthemonster/GratitudeJournal#163] [General]
Good day, 

Wanting to check in if I can help on Adding the needed to be able to Support Localization for Afrikaans?

Kind Regards
Jonathan

**Comment by alisonthemonster** [comment]
We would love to have your help here! There's a guide to translating here: https://presently-app.firebaseapp.com/translate.html

Feel free to open a PR or to just email us your translation and we can handle the rest!

**Comment by JonathanAspeling** [comment]
Great stuff - I've had a look at the guide and am continually impressed by how well maintained everything is - bravo - I'll sort the necessary 👏

**Comment by JonathanAspeling** [comment]
Done with Strings & the StoreListing , Bit beefed tonight - got repo setup on Local env.  Prompts next, will continue to work through it periodically and then hoping to take a stab at sorting inspirations too.

**Comment by JonathanAspeling** [comment]
Update here - we've got a civic tech bi-weekly hackathon type event on Thursday evening during which I will continue and perhaps even finish :D [Codebridge - Newlands](https://www.meetup.com/Codebridge/)

**Comment by JonathanAspeling** [comment]
Hay there update! Haven't given up here - recently had our first child - so been an interesting few weeks! Still on my radar :D

**Reply by alisonthemonster** [reply]
Congratulations on the new baby! Please take your time, no rush at all :)

**Comment by JonathanAspeling** [comment]
Hehe ended up taking a lot longer that I would have liked to get back to it! :D Needless to say I underestimated the life changes with our little one. 

I created the below Pull request for Afrikaans Localization <3 Cheers! 

https://github.com/alisonthemonster/Presently/pull/208
